Context creation in remote DAS is failing

```
======================================================================================== FAILURES ========================================================================================
___________________________________________________________________ TestRemoteDistributedAtomSpace.test_create_context ___________________________________________________________________

self = <hyperon_das.client.FunctionsClient object at 0x71f6a03ec250>, payload = {'action': 'create_context', 'input': {'name': 'my context', 'queries': None}}

    def _send_request(self, payload) -> Any:
        try:
            payload_serialized = serialize(payload)
    
            with sessions.Session() as session:
                response = session.request(
                    method='POST',
                    url=self.url,
                    data=payload_serialized,
                    headers={'Content-Type': 'application/octet-stream'},
                )
    
>           response.raise_for_status()

hyperon_das/client.py:36: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Response [500]>

    def raise_for_status(self):
        """Raises :class:`HTTPError`, if one occurred."""
    
        http_error_msg = ""
        if isinstance(self.reason, bytes):
            # We attempt to decode utf-8 first because some servers
            # choose to localize their reason strings. If the string
            # isn't utf-8, we fall back to iso-8859-1 for all other
            # encodings. (See PR #3538)
            try:
                reason = self.reason.decode("utf-8")
            except UnicodeDecodeError:
                reason = self.reason.decode("iso-8859-1")
        else:
            reason = self.reason
    
        if 400 <= self.status_code < 500:
            http_error_msg = (
                f"{self.status_code} Client Error: {reason} for url: {self.url}"
            )
    
        elif 500 <= self.status_code < 600:
            http_error_msg = (
                f"{self.status_code} Server Error: {reason} for url: {self.url}"
            )
    
        if http_error_msg:
>           raise HTTPError(http_error_msg, response=self)
E           requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://45.63.85.59:8080/function/query-engine

/usr/local/lib/python3.10/dist-packages/requests/models.py:1021: HTTPError

During handling of the above exception, another exception occurred:

self = <tests.integration.test_remote_das.TestRemoteDistributedAtomSpace object at 0x71f6a02023e0>, remote_das = <hyperon_das.das.DistributedAtomSpace object at 0x71f6a03ef2e0>

    def test_create_context(self, remote_das):
        context_name = 'my context'
>       context = remote_das.create_context(context_name)

tests/integration/test_remote_das.py:340: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
hyperon_das/das.py:775: in create_context
    return self.query_engine.create_context(name, queries)
hyperon_das/query_engines.py:580: in create_context
    return self.remote_das.create_context(name, queries)
hyperon_das/client.py:223: in create_context
    return self._send_request(payload)
hyperon_das/client.py:67: in _send_request
    das_error(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

exception = HTTPError('Please, check if your request payload is correctly formatted.', "'NoneType' object is not iterable")

    def das_error(exception: Exception):
        logger().error(str(exception))
>       raise exception
E       hyperon_das.exceptions.HTTPError: ('Please, check if your request payload is correctly formatted.', "'NoneType' object is not iterable")

hyperon_das/utils.py:23: HTTPError
----------------------------------------------------------------------------------- Captured log call ------------------------------------------------------------------------------------
ERROR    root:logger.py:41 ('Please, check if your request payload is correctly formatted.', "'NoneType' object is not iterable")
================================================================================ short test summary info =================================================================================
FAILED tests/integration/test_remote_das.py::TestRemoteDistributedAtomSpace::test_create_context - hyperon_das.exceptions.HTTPError: ('Please, check if your request payload is correctly formatted.', "'NoneType' object is not iterable")
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================== 1 failed, 30 passed in 98.67s (0:01:38) =========================================================================
```

--- a/hyperon_das/context.py
+++ b/hyperon_das/context.py
@@ -8,5 +8,5 @@ class Context:
 
     def __init__(self, context_node: dict[str, Any], query_answers: List[List[QueryAnswer]]):
         self.name = context_node['name']
-        self.handle = context_node['handle']
+        self.handle = context_node['_id']
         self.query_answers = query_answers

--- a/hyperon_das/das.py
+++ b/hyperon_das/das.py
@@ -67,7 +67,7 @@ class DistributedAtomSpace:
     def _create_context(
         self,
         name: str,
-        queries: Optional[List[Query]] = None,
+        queries: Optional[List[Query]] = [],
     ) -> Context:
         context_node = self.add_node({'type': Context.CONTEXT_NODE_TYPE, 'name': name})
         query_answer = [self.query(query, {'no_iterator': True}) for query in queries]
@@ -767,7 +767,7 @@ class DistributedAtomSpace:
     def create_context(
         self,
         name: str,
-        queries: Optional[List[Query]] = None,
+        queries: Optional[List[Query]] = [],
     ) -> Context:
         if self.system_parameters.get('running_on_server'):
             return self._create_context(name, queries)

--- a/tests/unit/test_cache_controller.py
+++ b/tests/unit/test_cache_controller.py
@@ -46,7 +46,7 @@ class TestCacheController:
     def test_add_context(self):
         controller = self._build_controller()
         context = Context(
-            {'name': 'blah', 'handle': 'h1'},
+            {'name': 'blah', '_id': 'h1'},
             [
                 [QueryAnswer({'handle': 'h1'}, None), QueryAnswer({'handle': 'h2'}, None)],
                 [QueryAnswer({'handle': 'h1'}, None), QueryAnswer({'handle': 'h3'}, None)],

--- a/tests/unit/test_context.py
+++ b/tests/unit/test_context.py
@@ -5,6 +5,6 @@ class TestContext:
     def test_creation(self):
         context_name = 'blah'
         context_handle = 'h'
-        context = Context({'name': context_name, 'handle': context_handle}, [])
+        context = Context({'name': context_name, '_id': context_handle}, [])
         assert context.name == context_name
         assert context.handle == context_handle

